manifests/011-prometheus-rules.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    name: samples-operator-alerts
+  name: samples-operator-alerts
+  namespace: openshift-cluster-samples-operator
+spec:
+  groups:
+  - name: SamplesOperator
+    rules:
+    - alert: SampleImageStreamImportsFailing
+      expr: max_over_time(sum(openshift_samples_failed_imagestream_import_total)[2h]) > 0
+      annotations:
+        message: |
+          Sample ImageStream imports are consistenly failing over 2 hours.
+    - alert: SamplesDegraded
+      expr: samples_operator_status_degraded == 1
+      annotations:
+        message: |
+          Samples could not be deployed and are in Degraded status.
```

Comment:
is this something other operators are doing? it seems slightly odd to have an alert for something that we already have explicit dashboards for.   but i can sort of see the rationale.

Reply:
yeah @ricardomaraschini 's analogous PR for image registry operator had this and I left it in when I copied some of his work .... @ricardomaraschini - did you see this in other operators, or came up with it on your own

similarly, I could be easily swayed either way ... keep/remove

Reply:
This alert was part of the ticket I was working on, not sure if others Operators implement it. I like the idea of having an alerts when something is not OK. A dashboard requires someone to be looking at it to see the problem.

Reply:
ok let's leave it in then, unless opinions change or new ones come in ... thanks  gentlemen